tendrl/alerting/handlers/__init__.py

```diff
@@ -88,6 +91,9 @@ def update_alert(self):
                 )
             )
 
+    def classify_alert(self):
```

Comment:
If its classify alert, shouldnt it take alert as an argument? I see alert.save() happening in specific implementation of classify_laert, then why we need alert.save() just before call to classify_alert? bit confusing to me.